std/format.d

```diff
@@ -1793,9 +1793,7 @@ unittest
         formatTest( to!(    const T)(5.5), "5.5" );
         formatTest( to!(immutable T)(5.5), "5.5" );
 
-        // bionic doesn't support lower-case string formatting of nan yet
-        version(CRuntime_Bionic) { formatTest( T.nan, "NaN" ); }
-        else { formatTest( T.nan, "nan" ); }
+        formatTest( T.nan, "nan" );
```

Comment:
Supported since Android 5.0, as the comments below indicate, so ripping these out.  If we ever decide to version on Android API numbers, we can go back and do something about this.